docs/dynamicproxy-introduction.md

```diff
@@ -9,7 +9,7 @@ DynamicProxy differs from the proxy implementation built into the CLR which requ
 To use Castle DynamicProxy you need the following environment:
 
 * one of the following runtimes installed
-  * .NET Framework 3.5 SP1+
+  * .NET Framework 4.0+
   * .NET Core 1.0+ (Castle Core 4.0+)
   * Silverlight 4.0+ (Silverlight support removed in Castle Core 4.0)
```

Comment:
Might as well remove Silverlight while you are there.

Reply:
Oh, right. I overlooked that one. Will remove.

Reply:
Done in 9e72dec6cd80c164dda1d634401cf4f06d38a79b.